qc_grader/challenges/quantum_explorers23/badge_5.py

```diff
@@ -166,8 +167,51 @@ def grade_badge5_ex15(answer15: List[int]) -> None:
 
 @typechecked
 def grade_badge5_code(qsvc: QSVC) -> None:
+
+    x_train = np.array([[1.25663706, 5.02654825, 3.76991118],
+       [3.76991118, 0.        , 0.62831853],
+       [4.08407045, 3.76991118, 1.88495559],
+       [0.62831853, 2.82743339, 2.51327412],
+       [0.9424778 , 5.96902604, 5.02654825],
+       [5.02654825, 2.51327412, 0.31415927],
+       [5.34070751, 0.62831853, 3.45575192],
+       [0.31415927, 3.45575192, 3.76991118],
+       [1.25663706, 4.39822972, 5.96902604],
+       [3.76991118, 1.25663706, 2.82743339],
+       [3.14159265, 5.34070751, 1.88495559],
+       [3.45575192, 0.31415927, 1.57079633],
+       [1.57079633, 0.        , 5.96902604],
+       [4.08407045, 3.14159265, 0.        ],
+       [1.57079633, 4.39822972, 0.9424778 ],
+       [0.        , 1.57079633, 0.31415927],
+       [5.02654825, 2.82743339, 4.71238898],
+       [2.51327412, 0.62831853, 0.62831853],
+       [3.14159265, 5.34070751, 1.88495559],
+       [5.65486678, 4.71238898, 4.08407045],
+       [0.        , 2.19911486, 3.45575192],
+       [0.62831853, 0.31415927, 3.45575192],
+       [5.65486678, 2.51327412, 5.65486678],
+       [2.19911486, 0.62831853, 5.02654825],
+       [5.34070751, 3.45575192, 4.39822972],
+       [4.08407045, 2.51327412, 4.08407045],
+       [0.62831853, 5.65486678, 4.08407045],
+       [5.96902604, 4.39822972, 0.62831853],
+       [5.96902604, 4.39822972, 5.96902604],
+       [0.        , 5.96902604, 4.08407045]])
+    
+    y_train = np.array([0, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 1, 1, 1, 1, 0, 0, 1, 1,
+       1, 0, 0, 0, 0, 1, 1, 0])
+
+    # Split the dataset into a training set and a test set
+    # 80% for training, 20% for testing”
+    train_size = int(0.8 * len(x_train))
+    x_test = x_train[train_size:]
+    y_test = y_train[train_size:]
```

Comment:
Only the test data (the last 20% entries) are needed for the grader. I put the whole data set and set the `train_size` (train/test split ratio) to slice the data because these are the coded used in the notebook and therefore it's easier to change if we ever decide to change the data set or `train_size`. @vabarbosa if you think it's better to only put the test data, I can make the changes.

Reply:
Decided to use only test data at the end: https://github.com/qiskit-community/Quantum-Challenge-Grader/pull/120/commits/e6c2d955868a1162db9de7d632b5838fe620529d

Reply:
yeah, i think that is better